hubspot/plugin.py

```diff
@@ -135,57 +478,61 @@ def create_contacts(contacts: List[HubSpotContact]) -> List[str]:
 
 
 @dataclass
-class HubSpotContactUpdate:
-    """The data required to update a HubSpot contact.
+class ContactDefaultPropertyUpdate:
+    """The data required to update a contact's default properties."""
 
-    Attributes:
-        id: The HubSpot ID of the contact to update.
-        updated_properties: A dictionary of property names and values to update for the
-            contact. Only properties that are to be updated need to be included.
+    contact_id: str
+    updated_properties: List[Tuple[HubSpotContactDefaultPropertyName, str]]
```

Comment:
Does 

```
Dict[HubSpotContactDefaultPropertyName, str]
```

work well here?

Seems much more natural as a dict rather than a List-Tuple

Reply:
Let me find out how we can use in plugins. 
```
@dataclass(frozen=True) 
class HubSpotContactDefaultPropertyName:
    name: Literal[...]
```